Dockerfile.openpilot_base

```diff
@@ -22,18 +22,23 @@ RUN cd /tmp && \
     rm -rf arm/ && \
     rm -rf thumb/nofp thumb/v6* thumb/v8* thumb/v7+fp thumb/v7-r+fp.sp
 
+ARG USER=batman
+RUN useradd -m -s /bin/bash -u 1000 $USER
```

Comment:
I would recommend also adding an `ARG` for the user ID. It makes it easy to build the container correctly if you log into your physical system as a user ID other than 1000.  The username used inside the docker container is irrelevant, what is important is the user ID matches the desired user ID outside the docker container (they can have different names).

Reply:
The dev container feature actually changes the UID of the "Batman" user to the UID of the host user

Reply:
This is kind of wild, I guess they try to re-build the container for you (or build a new container inheriting from the specified container?) if they detect the user ID inside the container doesn't match the user ID outside the container?!
https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user#_specifying-a-user-for-vs-code
```
However, UID/GID updates are only applied when the container is created and requires a rebuild to change.
```

Another section seems to indicate their magic maybe doesn't always work and so they provide an alternative example where you parameterize the user ID inside your Dockerfile:
https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user#_change-the-uidgid-of-an-existing-container-user